email_mailbox/src/lang/es.json

```diff
@@ -109,6 +109,28 @@
       }
     }
   },
+  "backup": {
+    "starting_backup": "Starting Backup...",
+    "saving_account": "Saving Account...",
+    "exporting_contacts": "Exporting Contacts...",
+    "saving_contacts": "Saving Contacts...",
+    "exporting_labels": "Exporting Labels...",
+    "saving_labels": "Saving Labels...",
```

Comment:
Not sure, but even es.json have to be in english?

Reply:
No. It needs translations.